nc-games-fe/src/Components/Single-Review.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchSingleReview } from "./utils";
+import ReviewCard from "./Review-Card";
+
+export const SingleReview = () => {
+    const {review_id} = useParams()
```

Comment:
Great use of use Params to ensure this page works if a user goes directly to the article through the link rather than just passing props down from the articles list 👍